syft/pkg/cataloger/python/parse_requirements.go

```diff
@@ -56,6 +56,11 @@ func parseRequirementsTxt(_ source.FileResolver, _ *generic.Environment, reader
 		version = strings.TrimFunc(version, func(r rune) bool {
 			return !unicode.IsLetter(r) && !unicode.IsNumber(r)
 		})
+
+		if name == "" || version == "" {
+			log.WithFields("path", reader.RealPath).Warnf("found empty package in requirements.txt line: %q", line)
```

Comment:
Should this be `Debugf`? I presume this will only be actionable if the user is scanning their own code

Reply:
good catch!